### PR TITLE
block metadata service for Alibaba Cloud

### PR DESCRIPTION
**Pull request by Lyt99**
block metadata service '100.100.100.200' on Alibaba Cloud

Signed-off-by: xiayu.lyt <xiayu.lyt@alibaba-inc.com>

### CODE REVIEW

File: go-controller/pkg/cni/OCP_HACKS.go [hunk 1]
```diff
@@ -25,6 +25,12 @@ var iptables4OnlyCommands = [][]string{
 	{"-A", "OUTPUT", "-p", "udp", "-m", "udp", "-d", "169.254.169.254", "!", "--dport", "53", "-j", "REJECT"},
 	{"-A", "FORWARD", "-p", "tcp", "-m", "tcp", "-d", "169.254.169.254", "!", "--dport", "53", "-j", "REJECT"},
 	{"-A", "FORWARD", "-p", "udp", "-m", "udp", "-d", "169.254.169.254", "!", "--dport", "53", "-j", "REJECT"},
+	// Block cloud provider metadata IP for Alibaba Cloud
```
Comment (abhat) [inline go-controller/pkg/cni/OCP_HACKS.go:28]: I think we need to figure out if OCP is running on Ali platform and then add these rules. I was thinking if we could somehow populate the infrastructure type from the infrastructure config object into a config map entry that we can read, and only add the rules for specific platform, that will be better. @trozet, @danwinship  wdyt?

Comment (kwoodson) [reply go-controller/pkg/cni/OCP_HACKS.go:28]: Any updates on this @abhat @trozet @danwinship?

Comment (abhat) [reply go-controller/pkg/cni/OCP_HACKS.go:28]: Closing in favor of https://github.com/openshift/ovn-kubernetes/pull/873

**Comment by Lyt99** [comment]
Closed for #873

**Pull request closed without merging**